extensions/toggl-track/package.json

```diff
@@ -148,6 +149,37 @@
       "description": "Whether or not to show the tags field in the create and update time entry forms.",
       "label": "Show tags",
       "default": true
+    },
+    {
+      "name": "timeEntriesLookbackNumber",
+      "type": "textfield",
+      "required": false,
+      "title": "Time Entries Lookback Amount",
+      "description": "Number of time units to look back when fetching time entries",
+      "placeholder": "Enter a number (e.g. 2)",
+      "default": "1"
+    },
+    {
+      "name": "timeEntriesLookbackUnit",
+      "type": "dropdown",
+      "required": false,
+      "title": "Time Entries Lookback Unit",
+      "description": "Time unit to use for lookback period",
+      "data": [
+        {
+          "title": "Days",
+          "value": "day"
+        },
+        {
+          "title": "Weeks",
+          "value": "week"
+        },
+        {
+          "title": "Months",
+          "value": "month"
+        }
+      ],
```

Comment:
While the UI looks nice, I think all we need is lookbackDays. Let's default to 14, and set a maximum of a reasonable number? I think we might end up overfetching and making loading speeds slow if we allow to fetch all time entries in history?

Reply:
Good point! I’ll update it to only include `lookbackDays` with a default of 14. I tried going back 60 days on my end and it works well, so I was thinking of setting that as the max limit. Does 60 days sound okay to you? That should help keep things performant while still offering a decent range.

Reply:
Sounds good!

Reply:
@franzwilhelm I've updated it, can you give it a look?